decidim-admin/app/views/decidim/admin/participatory_processes/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :title do %>
+  <h2><%= t "decidim.admin.titles.participatory_processes" %></h2>
+<% end %>
+
+<div class="actions title">
+  <%= link_to t("actions.new", scope: "decidim.admin", name: t("models.participatory_process.name", scope: "decidim.admin")), ['new', 'participatory_process'], class: 'new' %>
```

Comment:
Here https://github.com/codegram/decidim/pull/73/files#diff-9134f340108c9208ac6ae1a8c28e64c2R27 (code is not updated in this view)